src/widgets/study-with-us/constants.ts

```diff
@@ -1,6 +1,4 @@
-import { OptionItemProps } from '@/widgets/option-item/ui/option-item';
-
-export const studyOptions: OptionItemProps[] = [
```

Comment:
Why remove the type?

Reply:
part of the assignment described here https://github.com/orgs/rolling-scopes/projects/6/views/1?pane=issue&itemId=79815233

Reply:
I don't see any reason explicitly specifying types when TypeScript is smart enough to do it by itself


<img width="300" alt="Screenshot 2024-09-24 at 19 18 33" src="https://github.com/user-attachments/assets/8921e9ff-f08e-4fd4-a998-0cb9d7649a56">